### DISCUSSION TITLE
Możliwość tworzenia kart o dziwnej wartości losowości w edytorze

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#407] [Codding]
W edytorze można tworzyć karty o losowości wskazującej na możliwość wystąpienia wartości negatywnych, backend przyjmuje takie karty:

![image](https://user-images.githubusercontent.com/43492764/126631397-5dbfe138-4d69-4ea3-85b1-aafbb5826a77.png)

![image](https://user-images.githubusercontent.com/43492764/126632032-e685819e-8403-417d-99be-c59f54d717e7.png)

Stworzyć issue na teraz, czy to na inny już sprint?

**Comment by emkarcinos** [comment]
Tak, ale problem moim zdaniem leży na backendzie.
Taka losowść moim zdaniem powinna jak najbardziej być akceptowana, tylko przy obliczaniu np. damage, obetniemy te wartości do zera.

**Comment by emkarcinos** [comment]
Zrobiłem issue #410

**Reply by itchystone** [reply]
Super
